backend/collaboration-service/app/src/constants/language.ts

```diff
@@ -0,0 +1,14 @@
+export const LANGUAGES = {
+    JavaScript: "JavaScript",
+    Python: "Python",
+    "C#": "C#",
+    C: "C",
```

Comment:
why is the value a string instead of the `LANGUAGES` thing?

Reply:
The lang library is only used on frontend and the backend does not have knowledge of it